Singer.API/ClientApp/src/app/modules/admin/components/legalguardians/legalguardian-details/legalguardian-details.component.html

```diff
@@ -191,7 +191,7 @@ <h3 class="section-title">Zorggebruikers</h3>
                ></app-user-card>
             </mat-grid-tile>
          </mat-grid-list>
-      </div>
+         </div>
```

Comment:
Please fix indent.
Are you still not using VSCode correctly? Open root of repository, install recommended extensions, and this should be fixed by just saving the file ;)